### DISCUSSION TITLE
Can't find Encode's Logo

### DISCUSSION THREAD

**Discussion by omerXfaruq** [Kludex/uvicorn#1503] [Q&A] [answered]
Hello, would like to add Encode's logo on Gradio's README, but could not find it anywhere. Could you provide it somewhere, preferably in svg format, in high quality?
Logo [link](https://avatars.githubusercontent.com/u/19159390?s=200&v=4) from github 
cc: @tomchristie

**Comment by lovelydinosaur** [comment]
I think you might be able to find the logo on https://thenounproject.com (?)

**Reply by omerXfaruq** [reply]
Nope, couldn't find it [there](https://thenounproject.com/search/icons/?iconspage=1&q=encode).

I think having a Logos/Brand-assets page would be quite beneficial for Encode. Having a page the repo org or webpage would help people to find Encode's logos, and add the logo to their projects or webpages, or make use of it.

See
https://www.python.org/community/logos/

**Comment by br3ndonland** [comment, marked as answer]
A few things we could think about here:

- [Enable GitHub Discussions for the organization](https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization)
  - This is a question about Encode's main logo, not Uvicorn's logo, but we don't have a place on GitHub for these kinds of discussions
  - [Our Gitter](https://gitter.im/encode/community) can also be used for this purpose (search "logo" - there are some interesting comments), but it doesn't show up when people search on GitHub
- [Submit the logo to simple-icons](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon) (I'm happy to take care of a PR)
- Use the same logo on Encode's website ([the logo in the encode/encode.github.io repo is currently different](https://github.com/encode/encode.github.io/blob/bd6969adbb2fd8f7039fce82144480a5cf3ed835/img/icon.png)) (also happy to take care of a PR here)
- Consider storing brand assets in a central place, in both PNG and SVG formats (https://github.com/encode/uvicorn/discussions/1503#discussioncomment-2892081)
  - [encode/encode.github.io](https://github.com/encode/encode.github.io)?
  - A new repo, maybe named encode/logos?

In the meantime, it looks like you've [taken care of it on your end](https://github.com/gradio-app/gradio/blob/main/readme_files/encode.svg). Thanks!

**Reply by Kludex** [reply]
@br3ndonland Here you have: https://github.com/orgs/encode/discussions 🙏

Thanks @tomchristie 🙏

**Reply by omerXfaruq** [reply]
Oh it looks like I found a bug on Github:

I received this notification but it was redirecting to this link, which gave 404.

https://github.com/orgs/encode/discussions/1503#discussioncomment-3445708

And this is the correct link:
https://github.com/encode/uvicorn/discussions/1503#discussioncomment-3445708

It seems encode namespace changed into uvicorn after you enabled discussions in the repo.

**Reply by Kludex** [reply]
Yes.. It's been bothering me all day long.
